svg.js.d.ts

```diff
@@ -429,8 +434,14 @@ declare namespace svgjs {
         f: number;
         matrix: Matrix;
     }
+
+    interface Library {
+        Matrix: Matrix;
+    }
+
     export interface Matrix {
         (source: any): Matrix;
+        new(source: any): Matrix;

```

Comment:
You could also create a PR for his PR by forking his fork and merge it there then merge it here. However: Cherrypicking might be faster ^^